app/test_split_integer.py

```diff
@@ -2,20 +2,29 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    parts = split_integer(32, 6)
+
+    assert sum(parts) == 32
```

Comment:
```suggestion
    assert sum(parts) == split_integer(32, 6)
```
avoid using extra variables 

fix everywhere